src/ucp/wireup/address.c

```diff
@@ -1549,7 +1602,6 @@ ucs_status_t ucp_address_unpack(ucp_worker_t *worker, const void *buffer,
                              UCP_MAX_RESOURCES);
     ucp_address_entry_t *address_list, *address;
     uint8_t addr_flags;
-    ucp_object_version_t addr_version;
```

Comment:
why removed?

Reply:
it seems to complicated the code

Reply:
now we also have release version. are you suggesting to unpack them to tmp vars and then initialize address entry?

Reply:
yes, let's try and see if it improves the code